halo/attest/keeper/keeper.go

```diff
@@ -462,7 +462,46 @@ func (k *Keeper) latestAttestation(ctx context.Context, version xchain.ChainVers
 	}
 
 	if iter.Next() {
-		return nil, false, errors.New("multiple attestation found")
+		return nil, false, errors.New("multiple attestations" +
+			" found")
```

Comment:
nit: Why is this error msg broken in multiple lines?

Reply:
fixed, thanks